projects/plugins/jetpack/modules/masterbar/admin-menu/class-base-admin-menu.php

```diff
@@ -597,23 +604,14 @@ public function get_preferred_view( $screen, $fallback_global_preference = true
 	/**
 	 * Gets the identifier of the current screen.
 	 *
+	 * @deprecated Use the Dashboard_Switcher object instead.
+	 *
+	 * @todo: Replace the calls of this method with $this->switcher->get_current_screen().
+	 *
 	 * @return string
 	 */
 	public function get_current_screen() {
-		// phpcs:disable WordPress.Security.NonceVerification
```

Comment:
I've left the method in here to reduce the diff size and scope of the change - it acts now just as a wrapper. In the future we should remove the usage of this method and use directly the method from Dashboard_Switcher.